cares_lib/dynamixel/Servo.py

```diff
@@ -64,24 +77,30 @@ def __init__(self, port_handler, packet_handler, protocol, motor_id, LED_colour,
         self.max = max
         self.min = min
 
+        self.model = model
+
         self.total_results = 0
-        self.dxl_errors = 0
+        self.dxl_errors = 0 
+
+        self.addresses = addresses[self.model]
 
     @exception_handler("Failed to enable")
     def enable(self):
         self.disable_torque()
         self.set_control_mode(ControlMode.JOINT.value)# default set to joint mode to avoid moving on start up
-        self.limit_torque()
-        self.limit_speed()
-        self.enable_torque()
+        if self.model == "XL-320":
+            self.limit_torque()
+            self.limit_speed()
+            self.enable_torque()
         self.turn_on_LED()
 
     def state(self):
         current_state = {}
         current_state["position"] = self.current_position()
-        current_state["velocity"] = self.velocity_to_int(self.current_velocity())
-        current_state["load"]     = self.current_load()
- 
+        if self.motor_id != 10:
+            current_state["velocity"] = self.velocity_to_int(self.current_velocity())
+            current_state["load"]     = self.current_load()
```

Comment:
This shouldn't be here - the motor_id doesn't make sense being the hard-coded exception here

Reply:
There shouldn't be any exception to this method at all